### PR TITLE
Remove buggy apt update line

### PR DESCRIPTION
**Pull request by nessita**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,6 +44,7 @@ docker-bootstrap: clean
 
 venv: 
 	virtualenv -p python2 $(ENV)
+	$(ENV)/bin/pip install -U pip
```
Comment (facundobatista) [inline Makefile:47]: Update also `setuptools`, is something that brought us some issues being old after some changes they did last year.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -24,8 +24,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install Dependencies
       run: |
-        apt get update
-        apt install make
+        sudo apt install make
```
Comment (facundobatista) [inline .github/workflows/tests.yml:27]: I'm fine with this change, but note that in some systems that are very "minimal", you cannot install before update (it doesn't have the packages list, will not find `make`)

**Review by facundobatista — APPROVED** [review]
Nice :)

File: .github/workflows/tests.yml (hunk 2 above)
Comment (nessita) [reply .github/workflows/tests.yml:27]: Thanks! I added the update line. Also, thanks for the review!

File: Makefile (hunk 1 above)
Comment (nessita) [reply Makefile]: Thanks! Adding.

**Pull request merged**